src/api/spec/support/vcr.rb

```diff
@@ -4,7 +4,7 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'spec/cassettes'
   config.hook_into :webmock
-  config.default_cassette_options = { record: :once }
+  config.default_cassette_options = { record: :new_episodes }
```

Comment:
huh?

Reply:
This is me being sloppy. I'll go and change this and all the above